assisted-events-scrape/workers/cluster_events_worker.py

```diff
@@ -65,6 +70,9 @@ def store_events_for_cluster(self, cluster: dict) -> None:
             Anonymizer.anonymize_cluster(cluster)
             self._enrich_cluster(cluster)
 
+            if len(cluster["hosts"]) > MAX_HOSTS_COUNT:
+                raise ClusterTooLargeException(f"Cluster ID {cluster['id']} has too many hosts.")
```

Comment:
maybe something like this so we'll know how big they are:
```suggestion
                raise ClusterTooLargeException(f"Cluster ID {cluster['id']} has too many hosts ({len(cluster["hosts"])}>{MAX_HOSTS_COUNT}).")
```